deploy/operator/mirror_utils.sh

```diff
@@ -91,10 +91,12 @@ function mirror_file() {
   httpd_path="${2}"
   base_mirror_url="${3}"
 
+  local dir_name="$(dirname $(echo ${remote_url} | cut -d / -f 3-))"
```

Comment:
can you add a comment about how this parsing goes?

Reply:
This is in order to preserve the full path when mirroring the files. If not, when filenames are the same between different versions, they were overwritten

Extract the path from the URL by splitting by /, and keeping only fields 3 to end
```
$ echo https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.9/4.9.0/rhcos-live-rootfs.x86_64.img | cut -d / -f 3-
mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.9/4.9.0/rhcos-live-rootfs.x86_64.img
```

Get the directory name from the path using dirname
```
$ dirname mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.9/4.9.0/rhcos-live-rootfs.x86_64.img
mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.9/4.9.0
```

Running all linked together
```
$ dirname $(echo https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.9/4.9.0/rhcos-live-rootfs.x86_64.img | cut -d / -f 3-)
mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.9/4.9.0
```

Reply:
ack, that's more understandable right now
find a way to add comments (``# bla bla``) or separate that to local variables so that it will be more immediate to get